plugins/katello/nightly/user_guide/activation_keys/index.md

```diff
@@ -13,12 +13,14 @@ Activation Keys provide a mechanism to define properties that may be applied to
 - Subscriptions
 - Repository Enablement
 - Host Collections
+- System Purpose
 
 ## Definitions
 
 - [Content Host](/plugins/katello/{{ page.version }}/user_guide/content_hosts/index.html#what-are-content-hosts)
 - Host Collection - A statically defined group of Content Hosts.
 - Subscription - The right to receive the associated content from Katello.
+- System Purpose - When a system’s purpose is set, the entitlement server receives information that helps auto-attach a subscription that satisfies the intended use of the system.
```

Comment:
Let's phrase this: `Helps auto-attach find a subscription that satisfies the intended use of the system`

Reply:
Looks like you missed this in your recent update

Reply:
Ah. You want me to change the whole definition of System Purpose. My bad. I just changed the last part of the sentence.